lms/djangoapps/course_api/serializers.py

```diff
@@ -96,6 +101,26 @@ def get_blocks_url(self, course_overview):
         ])
         return self.context['request'].build_absolute_uri(base_url)
 
+    def get_price(self, course_overview):
+        price = None
+        site_currency = configuration_helpers.get_value('PAID_COURSE_REGISTRATION_CURRENCY')
+
+        if site_currency:
+            registration_price = CourseMode.min_course_price_for_currency(course_overview.id, site_currency[0])
+            currency = site_currency[0]
+        else:
+            registration_price, currency = CourseMode.min_course_price(course_overview.id)
+
+        if registration_price:
+            currency_symbol = dict(settings.CURRENCIES).get(currency, settings.PAID_COURSE_REGISTRATION_CURRENCY[0])
```

Comment:
I can not find where param `CURRENCIES ` is set up in the settings files. Do we need to specify it?

Reply:
These settings already set in production server. The task that related to it: https://youtrack.raccoongang.com/issue/CAMARA-51 
Also, I updated my comment in this task https://youtrack.raccoongang.com/agiles/104-17/105-19?issue=CAMARA-82 I wrote about check these setting.